src/ifp/src/InitFloorplan.cc

```diff
@@ -172,9 +172,13 @@ void InitFloorplan::initFloorplan(const odb::Rect& die,
     if (site) {
       // Destroy any existing rows.
       auto rows = block_->getRows();
-      for (dbSet<dbRow>::iterator row_itr = rows.begin(); row_itr != rows.end();
-           row_itr = dbRow::destroy(row_itr))
-        ;
+      for (dbSet<dbRow>::iterator row_itr = rows.begin();
+           row_itr != rows.end();) {
+        if (site != row_itr->getSite()) {
+          row_itr++;
+        }
```

Comment:
Shouldn't there be a continue here after row_iter++?  This will just delete whatever the next row is regardless of type (or fail if row_iter was just before end()).

Reply:
yes, thanks for catching that